paddlets/datasets/tsdataset.py

```diff
@@ -86,7 +86,7 @@ def load_from_dataframe(
         time_col: Optional[str] = None,
         value_cols: Optional[Union[List[str], str]] = None,
         freq: Optional[Union[str, int]] = None,
-        drop_tail_nan: bool = False,
+        drop_tail_nan: Optional[bool] = False,
```

Comment:
Line89 drop_tail_nan should Not be Optional, refers to  https://docs.python.org/3/library/typing.html#typing.Optional